src/Core/Mews.Fiscalizations.Core.Tests/Mews.Fiscalizations.Core.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
What happened here? :D 
Soem automatic change by VS?
Is it interchangeable as long as it's only 1 framework that's targeted?

Reply:
no, we changed the main projects but forgot the tests